modules/appeals_api/app/services/appeals_api/pdf_construction/stamper.rb

```diff
@@ -27,7 +27,7 @@ def date_stamp_path
       end
 
       def date_and_consumer_stamp_path
-        text = generate_text(10, @appeal.consumer_name)
+        text = generate_text(10, @appeal.consumer_name || '')
```

Comment:
I noticed (while creating example appeals in the console for manual testing) that if I hadn't specified a consumer name for the appeal, the `generate_text` method here would throw a 500 error by trying to read `nil.length` - I figured that even if this isn't happening to end users, it would be good to avoid a 500 error in any case